ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -191,12 +191,13 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CSISnapshot
```

Comment:
For 4.14, there is a known issue, we have to enable `MachineAPI` here.

Reply:
You mean we should keep `ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI` for 4.14 install but with name `aws-ipi-disconnected-sts-basecap-none-f28`? If so, how about drop `aws-ipi-disconnected-sts-basecap-none-f28` for 4.14 until the bug fix since the test checkpoint is for None cap?

Reply:
`MachineAPI` becomes a optional cap in 4.14, so once basecap is set to none, the machine-api would be disabled too, but that would lead the install failed, actually I am afraid there would be no real fix for such case, but just report some pre-check from installer or tell customers via document to guide them to configure the additional caps correctly in this case.

Reply:
> You mean we should keep ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI for 4.14 install but with name aws-ipi-disconnected-sts-basecap-none-f28

Right.

Reply:
ack, then i could update it in 4.14 install job config, and drop the test in 4.14-4.15 upgrade job. I will file another pr for 4.14-4.15 upgrade on such profile if MachineAPI would be a must add cap, is that ok for u?

Reply:
For sure, I can update the other jobs in another PR if that would introduce some more noise in this PR.

Reply:
I just updated but not touch the 4.14-4.15 upgrade job which does not enable the case test yet. The case need extra work if we need set additional cap for 4.14 install, so i will have another pr to deal with it. PTAL